src/components/SelectionList/index.tsx

```diff
@@ -27,18 +27,20 @@ function SelectionList<TItem extends ListItem>(props: BaseSelectionListProps<TIt
         };
     }, []);
 
+    const defaultOnScroll = () => {
+        // Only dismiss the keyboard whenever the user scrolls the screen
+        if (!isScreenTouched) {
+            return;
+        }
+        Keyboard.dismiss();
+    };
+
     return (
         <BaseSelectionList
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
             ref={ref}
-            onScroll={() => {
-                // Only dismiss the keyboard whenever the user scrolls the screen
-                if (!isScreenTouched) {
-                    return;
-                }
-                Keyboard.dismiss();
-            }}
+            onScroll={onScroll ?? defaultOnScroll}
```

Comment:
I'd love to do it that way but it's not possible :<.
To keep good animation performance we have to use `useAnimatedScrollHandler` from reanimated and it returns object instead of function(see animation in SearchPageBottomTab), so it can be called directly from UI thread in Animated component. Changing it again to function call would make it run on JS thread and animation would be laggy.
So we need to choose between having a dismissKeyboard call or our custom onScroll that's running animation.
Good thing we don't need to dismiss keyboard on SearchPage because it doesn't have text input

Reply:
I've tried the onScrollBeginDrag and onScrollEndDrag, but unfortunately none of these work on web. I've added the explaining comment.